core/src/test/java/org/apache/spark/unsafe/map/AbstractBytesToBytesMapSuite.java

```diff
@@ -622,6 +622,17 @@ public void initialCapacityBoundsChecking() {
     } catch (IllegalArgumentException e) {
       // expected exception
     }
+
+    try {
+      new BytesToBytesMap(
+              taskMemoryManager,
```

Comment:
Let's keep the indentation consistent

Reply:
ok,I will correct this indentation and propose JIRA.